spec/controllers/suggestion_controller_spec.rb

```diff
@@ -0,0 +1,307 @@
+require 'spec_helper'
+require 'rails_helper'
+
+require File.join('./app/controllers/application_controller')
+
+require File.join('./app/controllers/suggestion_controller')
+
+
+describe "SuggestionController" do
+
+  describe "#Test1:approve suggestion when having a topic in waitlist" do
+  
+    # Prepare for test
+    before(:each) do
+        @assignment_id = 711
+        @topic_id = 2864
+        @team_id = 27816
+        @assignment = Assignment.find_by_id(@assignment_id)
```

Comment:
So you are assuming that assignment 711 is in the testing db. Unfortunately it is not true on Travis CI. Travis CI only runs "db:seed" before running tests.